src/Runner.Worker/Program.cs

```diff
@@ -19,11 +19,16 @@ public static int Main(string[] args)
 
         public static async Task<int> MainAsync(IHostContext context, string[] args)
         {
+            Tracing trace = context.GetTrace(nameof(GitHub.Runner.Worker));
+            if (bool.TryParse(Environment.GetEnvironmentVariable("GITHUB_ACTIONS_RUNNER_ATTACH_DEBUGGER"), out bool waitForDebugger) && waitForDebugger)
```

Comment:
use `StringUtil.ConvertToBoolean()` instead of `bool.TryParse()`

Reply:
Do you have your listener side change ready? also the launch.task changes for VSCode?